openverse_catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -139,6 +134,29 @@ def base_repo_name(pr: dict):
     return pr["base"]["repo"]["name"]
 
 
+branch_protection_cache = defaultdict(dict)
+
+
+def get_branch_protection(gh: GitHubAPI, pr: dict) -> dict:
+    global branch_protection_cache
```

Comment:
Since `defaultdict` is a subclass of `dict` which is already mutable, I don't think the `global` reference is needed within this function.

Reply:
Done in https://github.com/WordPress/openverse-catalog/pull/642/commits/29d36664849d18b08cb36d19f5a36975dbf3d91a